app/models/service_template.rb

```diff
@@ -10,6 +10,20 @@ class ServiceTemplate < ApplicationRecord
     "storage"         => _("Storage")
   }.freeze
 
+
+  CATALOG_ITEM_TYPES = {
```

Comment:
Is there an object you can infer the name from? Or something like all subclasses of a base class? Who fills `prov_type`? Maybe this can be filled with some inflection methods https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inflector.rb ?

I also think this is rather a UI issue, they should infer the display string from `prov_type`